static/js/directives/chat.js

```diff
@@ -383,7 +383,7 @@ define(['jquery', 'underscore', 'text!partials/chat.html', 'text!partials/chatro
 								// Make sure we are not in group chat or the message is from ourselves
 								// before we beep and shout.
 								if (!subscope.isgroupchat && from !== sessionid) {
-									playSound.play("message1");
+									playSound.play("announce");
```

Comment:
For now, please revert. I understand the notion, but this requires more thought (providing ids for sounds).